examples/crm/src/companies/CompanyAside.tsx

```diff
@@ -85,7 +85,7 @@ export const CompanyAside = ({ link = 'edit' }: CompanyAsideProps) => {
                 <ReferenceField source="sales_id" reference="sales">
                     <FunctionField<Sale>
                         source="last_name"
-                        render={record =>
+                        render={(record?: Sale) =>
```

Comment:
This isn't necessary: since FunctionField already receives the Sale type, the render function is already typed